Description: Fix for Debian bug #613924 -- FTBFS in armel
 Fix for Debian bug #613924 -- ogre: FTBFS on armel - missing mach/machine.h
 .
 The patch just disables any attempt to proper detection of ARM features, as far
 as the author knows the information is not used for anything.
Author: Manuel A. Fernandez Montecelo <manuel.montezelo@gmail.com>
Bug-Debian: http://bugs.debian.org/613924
--- a/OgreMain/src/OgrePlatformInformation.cpp
+++ b/OgreMain/src/OgrePlatformInformation.cpp
@@ -40,10 +40,10 @@
 #include <signal.h>
 #include <setjmp.h>
 
-    #if OGRE_CPU == OGRE_CPU_ARM
-        #include <sys/sysctl.h>
-        #include <mach/machine.h>
-    #endif
+//    #if OGRE_CPU == OGRE_CPU_ARM
+//        #include <sys/sysctl.h>
+//        #include <mach/machine.h>
+//    #endif
 #endif
 
 // Yes, I know, this file looks very ugly, but there aren't other ways to do it better.
@@ -528,12 +528,12 @@
 //    done:
 //        fclose(cpuinfo); 
         
-        static char processor[257];
-        size_t s = sizeof processor;
-        static int mib[] = { CTL_HW, HW_MODEL };
-        if (sysctl (mib, 2, processor, &s, 0, 0) >= 0)
-            cpuID = processor;
-        else
+//        static char processor[257];
+//        size_t s = sizeof processor;
+//        static int mib[] = { CTL_HW, HW_MODEL };
+//        if (sysctl (mib, 2, processor, &s, 0, 0) >= 0)
+//            cpuID = processor;
+//        else
             cpuID = "Unknown ARM";
 
 #endif
